nixos/tests/nomad.nix

```
@@ -0,0 +1,55 @@
+import ./make-test-python.nix (
+  { lib, ... }: {
+    name = "nomad";
+    nodes = {
+      server = { config, pkgs, lib, ... }: {
+        networking = {
+          interfaces.eth1.ipv4.addresses = lib.mkOverride 0 [{
+            address = "192.168.1.1";
+            prefixLength = 16;
+          }];
+        };
+
+        environment.etc."nomad.custom.json".source =
+          (pkgs.formats.json { }).generate "nomad.custom.json" {
+            region = "universe";
+            datacenter = "earth";
+          };
+
+        services.nomad = {
+          enable = true;
+
+          settings = {
+            data_dir = "/var/lib/nomad";
+
+            server = {
+              enabled = true;
+              bootstrap_expect = 1;
+            };
+          };
+
+          extraSettingsPaths = [ config.environment.etc."nomad.custom.json".source ];
```

```suggestion
          extraSettingsPaths = [ "/etc/nomad.custom.json" ];
```

Otherwise it's the store path, making the `environment.etc` part above unnecessary.

The alternative is to inline it, but it's a good idea to test with a non-store path.

---

Alrighty.